<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommuSetupUI</class>
 <widget class="QWidget" name="CommuSetupUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CommuSetupUI</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="mainCommuTab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="testTab">
      <attribute name="title">
       <string>test</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>104</width>
         <height>41</height>
        </rect>
       </property>
       <property name="title">
        <string>Drive type</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QComboBox" name="driveTypeCombo"/>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="connectBtn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>connect</string>
       </property>
      </widget>
      <widget class="QPushButton" name="disconnectBtn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>disconnect</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>104</width>
         <height>41</height>
        </rect>
       </property>
       <property name="title">
        <string>Com port</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QComboBox" name="portNumCombo"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
